### DISCUSSION TITLE
Playlists with linked guides

### DISCUSSION THREAD

**Discussion by istoppedcaringat30** [iptv-org/community#608] [Q&A] [answered]
Hey all - forgive my ignorance on this.
I'm using Emby and have the playlist entered. I can browse channels and they load fine. But the guide data for them doesn't load.
Do I need to do anything additional for guide data or is it just supposed to work?

**Comment by BellezaEmporium** [comment, marked as answer]
You would need to generate the EPG separately, as we do not provide guides anymore.

Everything is at iptv-org/epg
